atc/worker/placement.go

```diff
@@ -1,31 +1,106 @@
 package worker
 
 import (
+	"errors"
+	"fmt"
 	"math/rand"
+	"strings"
 	"time"
 
 	"code.cloudfoundry.org/lager"
 	"github.com/concourse/concourse/atc/db"
 )
 
+type ContainerPlacementStrategyOptions struct {
+	ContainerPlacementStrategy string `long:"container-placement-strategy" default:"volume-locality" choice:"volume-locality" choice:"random" choice:"fewest-build-containers" choice:"limit-active-tasks" description:"Method by which a worker is selected during container placement. Multiple methods can be chained by +, e.g. volume-locality+fewest-build-containers"`
```

Comment:
Perhaps it'd be better to make this a `[]string` (and rename the field to `ContainerPlacementStrategies`, but keep the flag as is).

You'd then configure `concourse web` with:

```
--container-placement-strategy=volume-locality --container-placement-strategy=fewest-build-containers
```

or

```
CONCOURSE_CONTAINER_PLACEMENT_STRATEGY=volume-locality,fewest-build-containers
```

That way, it's consistent with other flags that accept multiple values

Reply:
Make sense. I just didn't realize `[]string` can be used. 😅